app/views/shared/_order_detail_action_form.html.haml

```diff
@@ -1,13 +1,18 @@
-- @facility_users ||= User.find_users_by_facility(current_facility)
+- @facility_users ||= User.find_users_by_facility(current_facility) || []
+
 %td.action-form
-  - if @facility_users.empty?
-    = select_tag :assigned_user_id, options_for_select([['Assign to...', nil], ['Unassign', 'unassign']]), :class => 'sync_select', :id => nil
-  - else
-    = select_tag :assigned_user_id, options_for_select(@facility_users.map{ |u| [u.full_name, u.id] }.unshift(['Assign to...', nil], ['Unassign', 'unassign'])), :class => 'sync_select', :id => nil
+  -# TODO: refactor/extract/I18n options building:
+  - assigned_user_options = @facility_users.map { |u| [u.full_name, u.id] }.unshift(["Assign to...", nil], ["Unassign", "unassign"])
+
+  = select_tag :assigned_user_id,
+    options_for_select(assigned_user_options), class: "sync_select", id: nil
+
 %td.action-form
-  %select{ :name => "order_status_id", :class => 'sync_select', :id => nil }
-    = options_for_select([['Status...', nil]])
-    - OrderStatus.non_protected_statuses(current_facility).each do |os|
-      = options_for_select([[os.name_with_level, os.id]])
+  %select{ name: "order_status_id", class: "sync_select", id: nil }
+    = options_for_select([["Status...", nil]])
+    - OrderStatus.non_protected_statuses(current_facility).each do |order_status|
+      = options_for_select([[order_status.name_with_level, order_status.id]])
+
+= render_view_hook "before_bulk_action_submit_button", facility: current_facility
```

Comment:
I feel like this might be better over the "Product" column. Right now it's pushing the submit button off the table.

![screen shot 2016-05-12 at 5 57 24 pm](https://cloud.githubusercontent.com/assets/1099111/15233060/07986576-186b-11e6-8d2a-f048da0ebc91.png)